models/nlp/pytorch/deberta/pretraining.py

```diff
@@ -0,0 +1,56 @@
+from transformers import DebertaTokenizer, DebertaModel
+import torch
+from transformers import DebertaConfig
+from transformers import (
+    DebertaConfig,
+    DebertaTokenizer,
+    DebertaForMaskedLM,
+    LineByLineTextDataset,
+    DataCollatorForLanguageModeling,
+    Trainer,
+    TrainingArguments
+)
+
+tokenizer = DebertaTokenizer.from_pretrained('microsoft/deberta-base')
+
+config = DebertaConfig()
+# !wget -c https://cdn-datasets.huggingface.co/EsperBERTo/data/oscar.eo.txt
+
+train_dataset = LineByLineTextDataset(
+    tokenizer=tokenizer,
+    file_path="/fsx/data/wikidemo/wiki_test",
+    block_size=128,
+)
+
+data_collator = DataCollatorForLanguageModeling(
+    tokenizer=tokenizer, mlm=True, mlm_probability=0.15
+)
+
+model = DebertaForMaskedLM(config=config)
+
+training_args = TrainingArguments(
```

Comment:
is the parameters here from paper?

Reply:
Except batchsize other parameters are from paper.